website_sale_order_type/views/assets.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="assets_frontend" inherit_id="website_sale.assets_frontend">
+        <xpath expr=".">
+            <script type="text/javascript"
+                    src="/website_sale_order_type/static/src/js/website_sale_order_type_tour.js"/>
```

Comment:
Why is this regular data instead of demo one? Put also a note for being moved to `assets_tests` + data on v13.

Reply:
Thanks. Changes made! Was that what you meant?

Reply:
It's still loaded as data instead of demo. In manifest, put the XML in demo key and move the file to demo folder.

Reply:
I think it was already like that: https://github.com/OCA/e-commerce/pull/454/files#diff-c638094d29e2fd858515ae4b86befdf3258a8b235bfad3842845768b76d1519cR18-R20
Right?

Reply:
I just need to move it to the `demo` folder

Reply:
Change done @pedrobaeza